packages/s/shepherd.js.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "shepherd.js",
+  "description": "Guide your users through a tour of your app.",
+  "keywords": [
+    "site tour",
+    "tour",
+    "tutorial",
+    "shepherd"
+  ],
+  "authors": [
+    "Adam Schwartz <adam.flynn.schwartz@gmail.com>",
+    "Zack Bloom <zackbloom@gmail.com>"
+  ],
+  "maintainers": [
+    "Robert Wagner <rwwagner90@gmail.com>",
+    "Chuck Carpenter <chuck@shipshape.io>"
+  ],
+  "license": "MIT",
+  "homepage": "https://shepherdjs.dev",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/shipshapecode/shepherd.git"
+  },
+  "npmName": "shepherd.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.@(js|css|map)"
+      ]
+    }
+  ],
+  "filename": "dist/js/shepherd.js"
```

```suggestion
  "filename": "js/shepherd.js"
```
`dist` is set as the baseDir, so we'll remove it when pulling files down.